firmware/library/L3_HAL/apds_9960.hpp

```diff
@@ -483,21 +466,14 @@ class Apds9960 : public Apds9960Interface
   }
 
  private:
-  static I2c i2c;
-  static I2cDevice<&i2c, 0x39, device::Endian::kLittle,
-                   Apds9960Interface::MemoryMap_t>
-      gesture;
-
   int8_t up_sensitivity_;
   int8_t down_sensitivity_;
   int8_t left_sensitivity_;
   int8_t right_sensitivity_;
   int16_t far_sensitivity_;
   int16_t near_sensitivity_;
```

Comment:
All of these private variables are defined once an never used. Make all of these static constexpr values.